samples/Exporters/Console/TestRedis.cs

```diff
@@ -31,46 +32,47 @@ internal static object Run(string zipkinUri)
             var connection = ConnectionMultiplexer.Connect("localhost:6379");
 
             // Configure exporter to export traces to Zipkin
-            using var tracerFactory = TracerFactory.Create(builder => builder
-                .UseZipkin(o =>
-                {
-                    o.ServiceName = "redis-test";
-                    o.Endpoint = new Uri(zipkinUri);
-                })
-                .AddInstrumentation(t =>
-                {
-                    var instrumentation = new StackExchangeRedisCallsInstrumentation(t);
-                    connection.RegisterProfiler(instrumentation.GetProfilerSessionsFactory());
-                    return instrumentation;
-                }));
-            var tracer = tracerFactory.GetTracer("redis-test");
+            using var openTelemetry = OpenTelemetrySdk.EnableOpenTelemetry(
+                builder => builder
+                    .UseZipkinExporter(o =>
+                    {
+                        o.ServiceName = "redis-test";
+                        o.Endpoint = new Uri(zipkinUri);
+                    })
+                    // TODO: Uncomment when we change Redis to Activity mode
+                    // .AddInstrumentation(t =>
```

Comment:
This example will be not generating any Redis traces now - please add a comment at the beginning of this class that this example is broken until the instrumentation switches to Activity.